### ISSUE TITLE
`SimBoot` always uses CI = 0.99

### ISSUE THREAD

**Issue by marklhc** [philchalmers/SimDesign#14]
Hi Phil,

Thank you for making the great `SimDesign` package. I'm exploring the `SimBoot` function but noticed that currently the argument `CI` did not get passed to the function, as the bootstrap CI always has 99% confidence level. Should be an easy fix for this. Thank you!

Best,

Mark

**Comment by philchalmers** [comment]
Good catch, what a silly mistake. Patch is available on the dev version now. Cheers.

**Comment by philchalmers** [comment]
Side note: `SimBoot()` has now been removed. I realized that assuming a normal confidence interval may not be the most optimal in simulation experiments due to the asymetry of some sampling distributions, so instead the CI methods have been placed within `runSimulation()` and `reSummarise()` via a `boot_method` argument. This now allows for empirical (`basic`), percentile, and normal approximation CIs (the previous default), where `basic` is probably the more reasonable default choice.
